src/xdp-context.c

```diff
@@ -239,16 +248,24 @@ xdp_context_export_portal (XdpContext             *context,
                         context);
     }
 
-  if (!g_dbus_interface_skeleton_export (skeleton,
-                                         context->connection,
-                                         DESKTOP_PORTAL_OBJECT_PATH,
-                                         &error))
-    {
-      g_warning ("Error: %s", error->message);
-      return;
-    }
+  if (g_dbus_interface_skeleton_export (skeleton,
+                                        context->connection,
+                                        DESKTOP_PORTAL_OBJECT_PATH,
+                                        &error))
+    g_debug ("Providing portal %s", name);
+  else
+    g_warning ("Error: %s", error->message);
```

Comment:
Probably needs a better error message. Otherwise we'll end up debugging something nice an incomprehensible like `Error: Permission Denied` in the future